### PR TITLE
feat(ci): add `pkg-pr` for branch `refactor/node`

### PR DESCRIPTION
**Pull request by RSS1102**
**Description:**

**BREAKING CHANGE:**

**Related issue (if exists):**

### CODE REVIEW

**Review by fu050409 — APPROVED** [review]
Require a approval for a test?

File: .github/workflows/pkg-pr.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Cache rust artifacts
+        uses: oxidecomputer/actions-rs_toolchain@oxide/master
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+
```
Comment (fu050409) [inline .github/workflows/pkg-pr.yml:28]: Action failed here https://github.com/farm-fe/farm/actions/runs/11178171789/job/31075215488?pr=1801, a `version` field should exsits. Perhaps try with `version: 9`?

**Review by fu050409 — CHANGES_REQUESTED** [review]

**Comment by RSS1102** [comment]
Please run again

**Review by fu050409 — APPROVED** [review]

**Comment by RSS1102** [comment]
Please  approved again

**Review by ErKeLost — APPROVED** [review]

**Pull request closed without merging**